NewEvaluation.html

```diff
@@ -23,56 +23,62 @@
 
 
 	<div class = "mainBlock">
-
+	<div class="spacing"></div>
 		<form >
-			<input type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
-			<input type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
-			<input type="date" name="Date" placeholder="dd/mm/yyyy" required/>
+			<input class="input-candidate" type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
+			<input class="input-candidate" type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
+			<input class="input-candidate" type="date" name="Date" placeholder="dd/mm/yyyy" required/>
 		</form>
 
+		<div class="littleSpacing"></div>
```

Comment:
yes true but how should I do the spacing for elements that don't support padding? Should I just try to make the size of the box of the element bigger and passionate the elements properly or to add a wrapper and add the needed distance or this is considered as artificial method as the one I used?